docs/topics/jvm/java-interop.md

```diff
@@ -46,6 +46,83 @@ fun calendarDemo() {
 Note that, if the Java class only has a setter, it isn't visible as a property in Kotlin because Kotlin doesn't support
 set-only properties.
 
+## Java synthetic property references
+
+> The feature is [Experimental](components-stability.md#stability-levels-explained). It may be dropped or changed at any time.
+> Use it only for evaluation purposes. We'd appreciate your feedback on it in [YouTrack](https://kotl.in/issue).
+>
+{type="warning"}
+
+Starting from Kotlin 1.8.20, you can create references to Java _synthetic properties_. For example, for such Java code:
+
+```
+public class Person {
+    private String name;
+    private int age;
+
+    public Person(String name, int age) {
+        this.name = name;
+        this.age = age;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public int getAge() {
+        return age;
+    }
+}
+```
+
+Kotlin always allowed to write `person.age`: `age` in this case is the synthetic property. Now, you can also 
+create references to `Person::age`, `person::age`. All the same works for `name` too.
+
+```
+val persons = listOf(Person("Jack", 11), Person("Sofie", 12), Person("Peter", 11))
+    Persons
+         // Call a reference to Java synthetic property:
+        .sortedBy(Person::age)
+         // Call Java getter via the Kotlin property syntax:
+        .forEach { person -> println(person.name) }
+}
+```
+
+### How to enable Java synthetic property references {initial-collapse-state="collapsed"}
```

Comment:
I plan to replace it with the link to a constant section about it when this PR is ready https://github.com/JetBrains/kotlin-web-site/pull/3465/files